internal/controllers/machine/machine_controller_status_test.go

```diff
@@ -675,13 +681,13 @@ func TestSetNodeHealthyAndReadyConditions(t *testing.T) {
 					Type:    clusterv1.MachineNodeHealthyV1Beta2Condition,
 					Status:  metav1.ConditionFalse,
 					Reason:  "SomeReason",
-					Message: "Node Ready: condition is False",
+					Message: "* Node.Ready: kubelet is NOT ready",
 				},
 				{
 					Type:    clusterv1.MachineNodeReadyV1Beta2Condition,
 					Status:  metav1.ConditionFalse,
 					Reason:  "SomeReason",
-					Message: "Some message (from Node)",
+					Message: "kubelet is NOT ready (from Node)",
```

Comment:
Hm, wondering if we should change that in some way

Reply:
> Hm, wondering if we should change that in some way (the "from Node" part)

Maybe we should just print NodeReady the exact same way as NodeHealthy (but only with the Node.Ready condition)

Current:
```yaml
  │           ├─NodeHealthy                                                                                                False   KubeletNotReady                 9m7s   * Node.Ready: container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady
│   │           │                                                                                                                                                           message:Network plugin returns error: cni plugin not initialized
│   │           ├─NodeReady                                                                                                  False   KubeletNotReady                 9m8s   container runtime network not ready: NetworkReady=false reason:NetworkPluginNotReady message:Network
│   │           │                                                                                                                                                           plugin returns error: cni plugin not initialized (from Node)
```